datastructure/queue_operate_all_composite/gen/large_max.cpp

```diff
@@ -0,0 +1,41 @@
+#include "random.h"
+#include <iostream>
+
+using namespace std;
+
+const int MOD = 998244353;
+const int Q_MAX = 200000;
+
+int main(int, char* argv[]) {
+    cin.tie(nullptr);
+    ios::sync_with_stdio(false);
+    constexpr char endl = '\n';
+
+    long long seed = atoll(argv[1]);
+    auto gen = Random(seed);
+
+    int q = Q_MAX;
+    cout << q << endl;
+
+    int length = 0;
+    for (int i = 0; i < q; i++) {
+        int t = gen.uniform(0, 2);
+        if (length <= Q_MAX / 2) {
+            t = 0;
+        }
+        cout << t;
+        if (t == 0) {
+            int a = gen.uniform(0, MOD - 1);
+            int b = gen.uniform(0, MOD - 1);
+            cout << ' ' << a << ' ' << b;
+            length++;
+        } else if (t == 1) {
+            length--;
+        } else {
+            int x = gen.uniform(0, MOD - 1);
+            cout << ' ' << x;
+        }
+        cout << endl;
```

Comment:
```suggestion
        cout << "\n";
```

出力が多いとかなりendlの速度が効いてくるので…

Reply:
実は `constexpr char endl = '\n';` してるので flush はされません :bow:
紛らわしくてよくなさそうなら修正しますが、どうしましょうか？

Reply:
あ、気付きませんでした　でしたら大丈夫です